Jenkinsfile_k8s

```diff
@@ -41,7 +41,8 @@ parallel(
   'updatecli': {
     updatecli(action: 'diff')
```

Comment:
```suggestion
    updatecli(action: 'diff', config: './updatecli/updatecli.d/daily.d')
    updatecli(action: 'diff', config: './updatecli/updatecli.d/weekly.d')
```

Reply:
Not needed anymore since I've kept udpatecli.d as root folder, available as default config :)

Reply:
Wrote those comments at the same time: [`74825b6` (#87)](https://github.com/jenkins-infra/docker-inbound-agents/pull/87/commits/74825b6dea6de5c15470d048b5696188d5d47dc9), good to you?